examples/ui/node_transform.rs

```diff
@@ -0,0 +1,388 @@
+                                        text: Text::from_section(
+                                            "<--",
+                                            TextStyle {
+                                                font: asset_server.load("fonts/FiraSans-Bold.ttf"),
+                                                font_size: 16.0,
+                                                color: Color::BLACK,
+                                            },
+                                        ),
+                                        ..default()
+                                    });
+                                });
+
+                            parent
+                                .spawn(ButtonBundle {
+                                    style: Style {
+                                        size: Size::all(Val::Px(50.)),
+                                        align_items: AlignItems::Center,
+                                        justify_content: JustifyContent::Center,
+                                        ..default()
+                                    },
+                                    background_color: Color::WHITE.into(),
+                                    ..default()
+                                })
+                                .insert(ScaleButton(-0.25))
+                                .with_children(|parent| {
+                                    parent.spawn(TextBundle {
+                                        text: Text::from_section(
+                                            "-",
+                                            TextStyle {
+                                                font: asset_server.load("fonts/FiraSans-Bold.ttf"),
+                                                font_size: 16.0,
+                                                color: Color::BLACK,
+                                            },
+                                        ),
+                                        ..default()
+                                    });
+                                });
+                        });
+
+                    parent
+                        .spawn(NodeBundle {
+                            style: Style {
+                                flex_direction: FlexDirection::Column,
+                                justify_content: JustifyContent::SpaceBetween,
+                                align_items: AlignItems::Center,
+                                size: Size::all(Val::Px(300.)),
+                                ..default()
+                            },
+                            background_color: Color::DARK_GRAY.into(),
+                            ..default()
+                        })
+                        .insert(TargetNode)
+                        .insert(NodeRotation::default())
+                        .insert(NodeScale::default())
+                        .insert(NodeTranslation::default())
```

Comment:
all the inserts should be grouped with the spawn (`spawn(Bundle{}).insert(A).insert(B)` => `spawn((Bundle{}, A, B)`)

Reply:
This reduces the table moves, and is thus significantly faster.